beta-src/src/components/ui/WDCommandButtons.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from "react";
+import { Stack } from "@mui/material";
+import WDCheckmarkIcon from "../svgr-components/WDCheckmarkIcon";
+import WDButton from "./WDButton";
+import VoteType from "../../types/Vote";
+import Vote from "../../enums/Vote";
+
+/**
+ * Define the voteState props passed to the function.
+ * Define the setVoteState props passed to the function.
+ */
+interface voteProps {
+  voteState: VoteType;
+  setState: (vote: Vote) => void;
```

Comment:
Based on your example in the conversation tab, wouldn't a better name for this be `toggleVote`?